README.md

```diff
@@ -159,46 +160,66 @@ To run e2e tests on projects affected by your changes:
 nx affected:e2e --base=origin/<base-branch>
 ```
 
-## Running The Application
+## Running The Full Stack Application
+
+Our project is a full stack application which need multiple parts to run properly. This section will detail how to run the project for different usecases.
 
-### Serving the application locally for development:
+### Serving the application locally for development
 
 1. Serve the front-end application:
    - `npm start mull-ui`
-2. Serve the back-end application (in a different tab):
+1. Serve the back-end application (in a different tab):
    - `npm start mull-api`
 
-### Building the application for production:
+### Running a local database for development
+
+You can serve the database locally for testing, and to avoid interfering with the dev/prod databases.
+
+1. Install [docker](https://docs.docker.com/get-docker/) and [MySQL Workbench](https://dev.mysql.com/downloads/workbench/) (or any similar SQL clients)
```

Comment:
I believe sequel pro did properly set up a local database on my machine

Reply:
It can connect correctly to the MySQL db right?

Reply:
Just want to share my experience: I had issues connecting to MySQL databases using sequel pro, but MySQL workbench worked 100% of the time

Reply:
Yeah im gonna keep the exact way it is right now, you can connect with mysqlbench, or other similar clients. No ned to complicate stuff